oak/server/dev/dev_oak_manager.cc

```diff
@@ -37,35 +32,42 @@ grpc::Status DevOakManager::CreateApplication(grpc::ServerContext* context,
                                               oak::CreateApplicationResponse* response) {
   std::string application_id = NewApplicationId();
   LOG(INFO) << "Creating app with with id: " << application_id;
+  auto runtime = absl::make_unique<OakRuntime>();
+  auto status = runtime->Initialize(request->application_configuration());
+  if (!status.ok()) {
+    return status.ToOtherStatus<grpc::Status>();
+  }
 
-  InitializeAssertionAuthorities();
-  runtime_->InitializeServer(
-      request->application_configuration(),
-      asylo::EnclaveServerCredentials(asylo::BidirectionalNullCredentialsOptions()));
+  grpc::ServerBuilder builder;
+  int port;
 
-  response->set_application_id(application_id);
-  response->set_grpc_port(runtime_->GetServerAddress());
-  return grpc::Status::OK;
-}
+  // Uses ":0" notation so that server listens on a free port.
+  builder.AddListeningPort("[::]:0", grpc::InsecureServerCredentials(), &port);
```

Comment:
I still think we should use asylo credentials even in this case. In fact perhaps the actual split is between enclave and non-enclave, but both should use asylo and the same key exchange protocol behind the scenes. I think that should work regardless of whether the server is in an Asylo application, but I may be wrong.